TradeItIosTicketSDK2/TradeItPortfolioAccountsTableViewManager.swift

```diff
@@ -34,7 +34,7 @@ class TradeItPortfolioAccountsTableViewManager: NSObject, UITableViewDelegate, U
 
         if accounts.count > 0 {
             let selectedAccount = selectedAccount ?? self.accounts[0]
-            let selectedAccountIndex = self.accounts.index(of: selectedAccount) ?? 0
+            let selectedAccountIndex = self.accounts.index(where: {$0.accountNumber == selectedAccount.accountNumber && $0.brokerName == selectedAccount.brokerName}) ?? 0
```

Comment:
we are using the account number and the brokerName to get the selectedAccount index